controller/src/test/java/io/pravega/controller/store/stream/ZkStreamTest.java

```diff
@@ -462,14 +462,14 @@ public void testTransaction() throws Exception {
                 context, executor).get();
```

Comment:
I don't see unit tests for the new methods introduced in `ZkStream`.

Reply:
The tests are introduced at base Stream test level (StreamTestBase) and it gets tested for all three store types.